### PR TITLE
v1/api: Add more logging for blueprint creation

### PR DESCRIPTION
**Pull request by ezr-ondrej**

### CODE REVIEW

File: internal/db/db_blueprints.go [hunk 1]
```diff
@@ -195,15 +195,15 @@ func (db *dB) InsertBlueprint(id uuid.UUID, versionId uuid.UUID, orgID, accountN
 	err = db.withTransaction(ctx, func(tx pgx.Tx) error {
 		tag, txErr := tx.Exec(ctx, sqlInsertBlueprint, id, orgID, accountNumber, name, description)
 		if tag.RowsAffected() != 1 {
-			return fmt.Errorf("%w, expected 1, returned %d", AffectedRowsMismatchError, tag.RowsAffected())
+			return fmt.Errorf("failed to insert blueprint: %w, expected 1, returned %d", AffectedRowsMismatchError, tag.RowsAffected())
 		}
 		if txErr != nil {
```
[the remark this answers is not in the document]
Comment (ezr-ondrej) [reply internal/db/db_blueprints.go:205]: Ah right, correct, let me switch the order

**Review by lzap — DISMISSED** [review]
Yeah that should do it.

**Comment by lzap** [comment]
Feel free to merge and do a followup, or rebase. I can do the followup if needed.

**Comment by ezr-ondrej** [comment]
I've added the info to all our db statements and reorginized error checks first there as well :)
IMO it's ready to get merged tomorrow :)

**Review by lzap — APPROVED** [review]
LGTM merge when you happy @avitova thanks.

**Review by avitova — APPROVED** [review]
LGTM:)

**Comment by lzap** [comment]
Thanks, merged for you.

**Pull request merged**